administrator/components/com_admin/postinstall/htaccessbrotli.php

```diff
@@ -8,15 +8,15 @@
  * @license     GNU General Public License version 2 or later; see LICENSE.txt
  *
  * This file contains post-installation message handling for notifying users of a change
- * in the default .htaccess file regarding setting the Content-Encoding header.
+ * in the default .htaccess file regarding Brotli compression.
  */
 
 // phpcs:disable PSR1.Files.SideEffects
 \defined('_JEXEC') or die;
 // phpcs:enable PSR1.Files.SideEffects
 
 /**
- * Notifies users of a change in the default .htaccess file regarding setting for brotli to prevent double compression
+ * Notifies users of a change in the default .htaccess file regarding Brotli compression
```

Comment:
```suggestion
 * Notifies users of a change in the default .htaccess file regarding setting for brotli to prevent double compression
```
Revert my change from conflict resolution